gh-pages/index.md

```diff
@@ -1,2 +1,2 @@
 * [Playground](playgrounds/tests/playground.html)
-* [Multi Playground](playgrounds/tests/multi_playground.html)
+* [Multi Playground](playgrounds/tests/multi_playground.html)
```

Comment:
Add your playground to this list.

Reply:
No new playground for reporter stuff, new playground will be added for functions

Reply:
Oh yes.